sdk/src/wallet/operations/balance.rs

```diff
@@ -34,9 +36,9 @@ where
         wallet_data: &WalletData,
     ) -> Result<Balance> {
         let network_id = self.client().get_network_id().await?;
-        let rent_structure = self.client().get_rent_structure().await?;
+        let storage_score_params = self.client().get_storage_score_parameters().await?;
         let mut balance = Balance::default();
-        let mut total_rent_amount = 0;
+        let mut total_storage_cost = 0;
```

Comment:
this variable name is still a bit off

Reply:
what do you think it should be?

Reply:
Idk, let's keep it a bit more, I need to refactor this file eventually anyway